projects/ng-event-plugins/src/types/longtap.event.ts

```diff
@@ -0,0 +1,13 @@
+interface LongtapEventInit extends EventInit {
+    clientX: number;
+    clientY: number;
+}
+
+export class LongtapEvent extends CustomEvent<{
+    clientX: number;
+    clientY: number;
+}> {
+    constructor(type: string, {clientX, clientY, ...eventInitDict}: LongtapEventInit) {
```

Comment:
Should type be hardcoded `longtap`?

Reply:
I don't know. This looks inconsistent with other events, such as `MouseEvent`, `TouchEvent`.

Reply:
Ok, let's keep it consistent. It's not really meant to be created outside plugin anyway.